src/DisasmViewer.cpp

```diff
@@ -281,6 +282,45 @@ void DisasmViewer::paintEvent(QPaintEvent* e)
 	visibleLines -= partialBottomLine;
 }
 
+void DisasmViewer::copyCodeToClipboard() const
+{
+    QClipboard *clipboard = QApplication::clipboard();
+	QString buffer;
+	
+	// calculate lines while writing
+	int currentLine = 0;
+	int y = frameT;
+
+	const DisasmRow* row;
+	bool displayDisasm = memory != nullptr && isEnabled();
```

Comment:
> Just a suggestion. Maybe you could instead offer a dialog where the user fills-in start and end address (you can pre-fill these values with the top and bottom instruction in the current window) (possibly allow to choose between clipboard and filename) and then press a button to generate the disassembly. This allows to generate disassembly of a much larger program, not just what fits on screen.

I think we can do it in the next iteration. That would require the program to check memory bounds and fetch more if not found since the debugger keeps tabs of small windows each time.